roles/ood_user_reg/tasks/main.yml

```diff
@@ -66,3 +66,29 @@
 - name: Build the updated Apache config
   command: /opt/ood/ood-portal-generator/sbin/update_ood_portal
   ignore_errors: yes
+
+- name: Create RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo id -u {{ RegUser_app_user }} &>/dev/null || sudo useradd -c "{{ RegUser_app_user_full_name }}" {{ RegUser_app_user }}'
```

Comment:
Make sure to use the ansible user create module.

https://docs.ansible.com/ansible/latest/modules/user_module.html

Reply:
Additionally, this user should probably be created as a `system` user, i.e. `system: yes` as better clarifies the role of the user and will assign it an id from the system user range vs the higher range used by normal users.

Reply:
But i think we do not want this user as `system` user, right?